### ISSUE TITLE
errors accessing ccessToken

### ISSUE THREAD

**Issue by walter-grace** [HumeAI/hume-evi-next-js-starter#5]
✓ Compiled in 1649ms (1787 modules)
 ⨯ app/page.tsx (12:11) @ Page
 ⨯ Error
    at Page (./app/page.tsx:23:15)
digest: "279398420"
  10 |
  11 |   if (!accessToken) {
> 12 |     throw new Error();
     |           ^
  13 |   }
  14 |
  15 |   return (
 ⨯ app/page.tsx (12:11) @ Page
 ⨯ Error
    at Page (./app/page.tsx:23:15)
digest: "279398420"
  10 |
  11 |   if (!accessToken) {
> 12 |     throw new Error();
     |           ^
  13 |   }
  14 |
  15 |   return (
 ⨯ app/page.tsx (12:11) @ Page
 ⨯ Error
    at Page (./app/page.tsx:23:15)
digest: "279398420"
  10 |
  11 |   if (!accessToken) {
> 12 |     throw new Error();
     |           ^
  13 |   }
  14 |
  15 |   return (

**Comment by jaid-monwr** [comment]
Hi @walter-grace, I was facing the same issue and I FIXED the problem.

Solution:
1. Open the file getHumeAccessToken.ts under utils folder
2. Rename the variable clientSecret to secretKey in:

```
const accessToken = await fetchAccessToken({
    apiKey: String(process.env.HUME_API_KEY),
    secretKey: String(process.env.HUME_CLIENT_SECRET),
  });
```
  
 That should fix your problem.

**Comment by twitchard** [comment]
Fixed in #8

**Comment by santjosie** [comment]
I am getting this error with the latest version of the template app. I have downloaded the app locally. Put in the API keys in a .env file. I am running the app locally.

The fetchAccessToken function from the hume package is not retrieving a valid access token - even though the API key and the secret key are being fetched correctly from the .env file.

The key for secret key in the JSON parameter is secretKey.

Is there any way to catch the error message?
